mne/decoding/tests/test_time_gen.py

```diff
@@ -156,6 +156,8 @@ def test_generalization_across_time():
     scores = gat.score()
     assert_true(scores is gat.scores_)
     assert_equal(np.shape(gat.scores_), (15, 1))
+    assert_true(np.all([tim for ttime in gat.test_times_['times']
+                        for tim in ttime] == gat.train_times_['times']))
```

Comment:
we use it often, `from numpy.testing import ...` -- take a look at our other tests and you'll see `assert_array_equal`, `assert_array_almost_equal`, `assert_allclose`, etc. from `numpy.testing` which are all useful.